python/lsst/ts/ofc/ctrlIntf/OFCCalculationOfLsstFam.py

```diff
@@ -0,0 +1,44 @@
+# This file is part of ts_ofc.
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+from lsst.ts.ofc.Utility import InstName
+from lsst.ts.ofc.ctrlIntf.OFCCalculation import OFCCalculation
+from lsst.ts.ofc.ctrlIntf.FWHMToPSSN import FWHMToPSSN
+
+
+class OFCCalculationOfLsstFam(OFCCalculation):
+    """The concrete child class of OFCCalculation of the LSST."""
+
+    def __init__(self, state0Dof=None):
+        """Construct an OFC calculation of LSST full-array mode (FAM).
+
+        Parameters
+        ----------
+        state0Dof : dict, optional
+            State 0 DoF dictionary. If None (=default), the instrument's
+            default will be used. See
+            :lsst:ts:ofc:`OptCtrlDataDecorator.getState0FromDict` for format
+            details.
+        """
+
+        super(OFCCalculationOfLsstFam, self).__init__(
```

Comment:
Python3 comes with slightly shorter version of super: super().__init__(, name of the class not needed.

Reply:
Fixed in the commit: c90538b. Thanks!